src/entity/Comment.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace Projet5\entity;
+
+/**
+ * Comment
+ */
+class Comment
+{
+    private $id;
+    private $contents;
+    private $date_comment;
+    private $publish;
+    private $id_bpf_blog_posts;
+    private $id_bpf_users;
+
+    /**
+     * Method getId
+     * 
+     * Get value of idComment
+     *
+     * @return void
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Method setId
+     * 
+     * Set value of idComment
+     *
+     * @param int $id
+     *
+     * @return void
+     */
+    public function setId($id): void
+    {
+        $this->id = $id;
```

Comment:
```suggestion
        $this->id = $id;
        return $this;
```

Reply:
return $this;    

Ajouté à tous les setters